doc/tech/usecases.md

```diff
@@ -0,0 +1,194 @@
+
+Extend CNTT's Scope beyond the Regional and National Data Center cloud infrastructures to the Edge. The Edge is a disruptive use case where CNTT can add value especially as there are a number of scattered initiatives under various Standard Development Organisations (SDO) and open source communities."
+
+Edge Stream related activities :
+
+To Harmonize the work under Standard Development bodies or Open source communities
+To build Common Cloud infrastructure based on CNTT principles that can be consum ed by any operator
+To build Cloud infrastructure that can scale over hundreds of thousands of nodesand cover the Edge Telco use cases that can help operators to monetize the NFV/SDN journey
+Modify the existing RM and RA-s so that they are aligned with the edge requirements
+
+<a name="5.1.3"></a>
+### 5.1.3 Approach & Scope
+
+All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles (if needed) specific to Edge use cases such as profiles required for acceleration technologies 
+-	Define resource requirements in terms of compute, storage and networking; new concepts, such as hyper converged infrastructure, can     be introduced
+-	Define different Architecture models as no one size fits all"
+-	
+
+Out of Scope 
+
+-	The API that can be exposed to 3rd party application developer 
+-	The Architecture of VNF/CNF ( CNTT Guideline only )
+-	Exact location of Edge deployment will vary by Operator 
+-	latency may be vary based on operator transmission type (Microwave,Fixed),a well Transmission network reachability and architecture (Backhaul,front haul, Mid haul)
+-	Onboarding of VNF/CNF on edges cloud infrastructure.
+
+<a name="5.1.4"></a>
+### 5.1.4 Principle
+
+This section introduces some principles that should be followed during the definition and development of the Edge scope to be covered in CNTT Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites.
+
+A main principle is that we shall not re-define a new branch of CNTT and avoid to re-inventing what other organizations already have . we are following the same principles that defined in existing [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles), the [Reference Architecture Principles](../ref_arch#principles) and the Network Principles.
+
+CNTT believe that Edge is unique in terms of infrastructure requirement ,  Implementation and deployment . that why There are some additional principle be defined covering a specific to the edge.
```

Comment:
Suggest:

CNTT believes that Edge computing is unique in terms of infrastructure requirements,  implementation and deployment, and that's why there some additional principles specific to the edge need to be defined.

Reply:
done